test/controllers/api/accounts_controller_test.rb

```diff
@@ -111,12 +111,10 @@ class Api::AccountsControllerTest < ActionController::TestCase
     end
 
     test 'returns account for matching tag without cleartext username / password' do
-
       login_as(:bob)
+      get :index, params: { 'tag': 'www.tag.ch' }, xhr: true
```

Comment:
try to avoid using existing domain names for test data.
use www.example.com vor example

this should probably also not be part of this pull request/commits. please move